src/digital_identity_service/share.qr.code.fetch.result.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const Validation = require('../yoti_common/validation');
+
+const ShareSessionCreateResult = require('./share.session.create.result');
+
+/**
+ * The fetch share QR code result
+ *
+ * @class ShareQrCodeFetchResult
+ */
+module.exports = class ShareQrCodeFetchResult {
+  /**
+   * @param {Object} response
+   */
+  constructor(response) {
+    Validation.isString(response.id, 'QR code ID');
+    this.id = response.id;
+
+    Validation.isString(response.expiry, 'Expiry');
+    const expiryDate = new Date(response.expiry);
+    if (expiryDate.toString() === 'Invalid Date') throw TypeError('Expiry must be a date like string');
+    this.expiry = expiryDate;
```

Comment:
You can use `Validation.isStringDate`.

Reply:
Good shout!
Could you make the change @abdalmajeed-yoti  (given M is absent today)